src/index.js

```diff
@@ -203,14 +203,14 @@ function shouldLoadFile(filename) {
  * @returns Babel plugin object with Program and CallExpression visitors
  */
 function addMetadata({ types: t }) {
+  if (!process.env.BABEL_TEST_METADATA) {
+    return {};
+  }
+
   return {
     name: 'addMetadata',
     visitor: {
       Program(babelPath, state) {
-        if (!state.opts.enabled) {
```

Comment:
can we leave this in case we're able to get the options working?

Reply:
I think we should leave this out to avoid any confusion about its presence. We can add it back in as a follow up along with a working options implementation.